src/webpack/loaders/devSiteExample.js

```diff
@@ -22,7 +22,7 @@ const loader = async function loader(content) {
 
   if (cssFileName !== undefined) {
     try {
-      this.resolve('', cssFileName, async () => {
+      this.resolve(this.context, cssFileName, async () => {
```

Comment:
Not totally sure how this worked before but we want to resole the cssFile in context of `this` file, not in context of the current directory.

Reply:
I always hate when that happens. Find something that should be a bug and not sure how it ever worked.